services/core-web/src/components/noticeOfWork/applications/process/ProcessPermit.js

```diff
@@ -296,9 +295,13 @@ export class ProcessPermit extends Component {
       <div>
         <div className={this.props.fixedTop ? "view--header fixed-scroll" : "view--header"}>
           <div className="inline-flex block-mobile padding-md">
-            <h2>
-              Process Permit
-              <CoreTooltip title="This page allows you to review the progress of the Notice of work and record decisions. You can also generate any decisions letters once a decision is made." />
+            <h2 className="tab-title">
+              <Popover
+                placement="topLeft"
+                content="This page allows you to review the progress of the Notice of work and record decisions. You can also generate any decisions letters once a decision is made."
```

Comment:
"decision letters" or "decisions letters"?

Reply:
I removed the last sentence as it doesn't seem accurate - This page allows you to review the progress of the Notice of Work application and record decisions.